src/OpenLoco/OpenLoco.cpp

```diff
@@ -857,12 +858,50 @@ namespace OpenLoco
         }
     }
 
-    static void sub_496A84(int32_t edx)
+    // 0x00496A84
+    static void updateSnowLine(int32_t currentDayOfYear)
     {
-        // This is responsible for updating the snow line
-        registers regs;
-        regs.edx = edx;
-        call(0x00496A84, regs);
+        auto* climateObj = ObjectManager::get<ClimateObject>();
+
+        uint8_t al = climateObj->var_02;
```

Comment:
loco tools has climate 02 as the first season.

Reply:
I noticed that, too, but it doesn't make sense. It's being subtracted from the ordinal day.